### PR TITLE
/recent shows up to 5 recent completed games

### PR DESCRIPTION
**Pull request by MatthewMerrill**

### CODE REVIEW

File: src/main/java/io/chrislowe/discordle/Main.java [hunk 1]
```diff
@@ -132,6 +135,14 @@ public void registerCommands(GatewayDiscordClient gateway) {
                 .createGlobalApplicationCommand(applicationId, userCommandRequest)
                 .subscribe();
 
+        ApplicationCommandRequest recentCommandRequest = ApplicationCommandRequest.builder()
+            .name("recent")
+            .description("View recent games!")
+            .build();
+        gateway.getRestClient().getApplicationService()
+            .createGuildApplicationCommand(applicationId, 190983259490877440L, recentCommandRequest)
```
Comment (HelloChrisLowe) [inline src/main/java/io/chrislowe/discordle/Main.java]: I think we want to make this a global app command

**Review by HelloChrisLowe — CHANGES_REQUESTED** [review]
Cool change, thanks!

File: src/main/java/io/chrislowe/discordle/Main.java (hunk 1 above)
Comment (MatthewMerrill) [reply src/main/java/io/chrislowe/discordle/Main.java]: LOL yes we do

Comment (MatthewMerrill) [reply src/main/java/io/chrislowe/discordle/Main.java]: Done

**Comment by HelloChrisLowe** [comment]
LGTM, thanks!

**Pull request merged**